### DISCUSSION TITLE
patched

### DISCUSSION THREAD

**Discussion by chuangjonathan123** [3kh0/ext-remover#162] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
hi they patched both ingot and the gui is there any other ways? Please reply thank you!

**Comment by ghost** [comment]
hey, this happened for me too...

**Comment by 3kh0** [comment, marked as answer]
What version are you on? Is bookmarklets blocked? Please explain more

**Reply by chuangjonathan123** [reply]
i just recently updated my chromebook, so I think thats the problem/ For every method, ingot , gui, it will show it as disbaled but every time I refresh it it enables it agian. This hnever happened before I updated it Hope you understand more!

**Reply by ehunt34** [reply]
Both LTS 102 and 106 have patched this. You can roll back with an image from chrome100.dev then recover your chromebook.

**Reply by chuangjonathan123** [reply]
ye ik but chrome100.dev is blocked on my Chromebook. Is there a way to recover versions without using that website? Thank you!

**Reply by ehunt34** [reply]
You can access another computer, or find an image somewhere else. Searching for the name directly helps. Search up "chromeos_14526.89.0_board_recovery_stable-channel_mp-v7.bin.zip", replacing board with what your devices board name is, listed at https://www.chromium.org/chromium-os/developer-information-for-chrome-os-devices/ . You can then use the recovery extension, use a local image, then use this with it.

**Reply by ghost** [reply]
NO DOWNGRADING! The sys admin WILL be notified if a chromebook is downgraded, and the school can press charges on you! You can SERIOUSLY go to prison for this.

**Reply by ghost** [reply]
@3ch0 are*

**Reply by Fallenmoon8080** [reply]
How do you know that SYSadmin is notified upon downgrading. Don't want to downgrade, just how do you know?

**Reply by ghost** [reply]
dude. the chromebooks are enrolled. they can check all the device info, including if you are unenrolled/downgraded

**Reply by LuxeCommands** [reply]
> @3ch0 are*

💀

**Reply by ghost** [reply]
@LuxeCommands problem?

**Comment by chuangjonathan123** [comment]
Can anyone help me pls my school blocked everything including the option of downgrading, any other ways? thx so much

**Reply by md4r2** [reply]
u could use a proxy

**Reply by chuangjonathan123** [reply]
whats some good unblocked proxys?

**Reply by Theblackmamba824** [reply]
search up b central it is a google sites with many proxy links
